### PR TITLE
use `run_shell_cmd` in custom easyblock for WPS

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _tcsh-6.22.03-GCCcore-10.2.0.eb_ 
 * **SUCCESS** _tcsh-6.22.04-GCCcore-10.3.0.eb_ 
 * **SUCCESS** _time-1.9-GCCcore-11.3.0.eb_ 
 * **SUCCESS** _WRF-4.4-foss-2022a-dmpar.eb_ 
 * **SUCCESS** _WPS-4.4-foss-2022a-dmpar.eb_ 
 * **SUCCESS** _time-1.9-GCCcore-10.2.0.eb_ 
 * **SUCCESS** _time-1.9-GCCcore-10.3.0.eb_ 
 * **SUCCESS** _WRF-3.9.1.1-foss-2020b-dmpar.eb_ 
 * **SUCCESS** _WRF-4.2.2-foss-2020b-dmpar.eb_ 
 * **SUCCESS** _WRF-4.3-foss-2021a-dmpar.eb_ 
 * **SUCCESS** _WPS-3.9.1-foss-2020b-dmpar.eb_ 
 * **SUCCESS** _WPS-4.2-foss-2020b-dmpar.eb_ 
 * **SUCCESS** _WPS-4.3.1-foss-2021a-dmpar.eb_ 

Build succeeded for 13 out of 13 (4 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/f7aee0018a1099368f66f8bdb4f5c6b4 for a full test report.

File: easybuild/easyblocks/w/wps.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ##
-# Copyright 2009-2024 Ghent University
+# Copyright 2009-2023 Ghent University
```
Comment (branfosj) [inline easybuild/easyblocks/w/wps.py:2]:
```suggestion
# Copyright 2009-2024 Ghent University
```

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _WPS-4.3.1-foss-2021a-dmpar.eb_ 
 * **SUCCESS** _WPS-4.4-foss-2022a-dmpar.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3564.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/de204100c58f4fd30b9dcf213adbd9fa for a full test report.

**Pull request merged**